ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -48,8 +48,9 @@
     private final String[] args;
     private String objectName;
     private String proxy;
-    private int start;
-    private int end;
+    //TODO re-add these calls when we have support for partial file gets in the helper functions
```

Comment:
I would just outright remove these.  I don't think we have any plans to add partial gets back at any time.  And they are supported in the helper functions now.